scripts/install/trackarr.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+#Trackarr installer
+
+pvryaml="/opt/trackarr/pvr.yaml"
+
+_install() {
+    trackarr_releases=$(curl -s "https://gitlab.com/api/v4/projects/cloudb0x%2Ftrackarr/releases/" | jq '.[0].assets.links[] | {url: .url} ') || {
+        echo_error "Failed to query GitLab"
+        exit 1
+    }
+    case $(_os_arch) in
+        "amd64") dlurl="$(echo "${trackarr_releases}" | jq -r '.|select(.url | contains("linux_amd64"))|.url')" ;;
+        "arm64") dlurl="$(echo "${trackarr_releases}" | jq -r '.|select(.url | contains("linux_arm64"))|.url')" ;;
+        "armhf") dlurl="$(echo "${trackarr_releases}" | jq -r '.|select(.url | contains("linux_armhf"))|.url')" ;;
+        *)
+            echo_error "Arch not supported"
+            exit 1
+            ;;
+    esac
+    echo_progress_start "Downloading trackarr and extracting"
+    if ! wget "$dlurl" -O /tmp/trackarr.tar.gz >> $log 2>&1; then
+        echo_error "Failed to download"
+        exit 1
+    fi
+    rm -rf /opt/trackarr
+    mkdir -p /opt/trackarr
+    tar -C /opt/trackarr -xzf /tmp/trackarr.tar.gz >> $log
+    echo_progress_done "Trackarr downloaded and extracted"
+
+    useradd --system trackarr -d /opt/trackarr
+    chown -R trackarr:trackarr /opt/trackarr
+    /opt/trackarr/trackarr >> $log 2>&1
+
+}
+
+_nginx() {
+    if [[ -f /install/.nginx.lock ]]; then
+        echo_progress_start "Configuring nginx for trackarr"
+        /etc/swizzin/scripts/nginx/trackarr.sh
+        systemctl reload nginx
+        echo_progress_done "Nginx configured"
+    else
+        #shellcheck source=sources/functions/utils
+        . /etc/swizzin/sources/functions/utils
+        user=$(_get_master_username)
+        pass=$(_get_user_password "$user")
+        sed -i "/^server:*/a \ \ pass: $pass" /opt/trackarr/config.yaml
```

Comment:
We should not be storing plain-text passwords in world readable files. We should ensure the permissions on the `/opt/trackarr` directory are tight to prevent password leakage in the event of node sharing.

Reply:
good catch, resolved!